penguin/tools/browser_harness_tools.py

```
@@ -0,0 +1,860 @@
+        self,
+        full: bool = False,
+        max_dim: Optional[int] = None,
+        output_dir: Optional[str] = None,
+    ) -> Dict[str, Any]:
+        directory = Path(
+            output_dir
+            or self.config.get("screenshot_dir")
+            or Path.cwd() / "screenshots"
+        )
+        directory.mkdir(parents=True, exist_ok=True)
+        timestamp = datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
```

Ruff (0.15.12)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)